src/helpers/setLocal.js

```diff
@@ -26,5 +26,5 @@ export default (req) => {
 
   const localeData = require('../locale/' + currentLocal + '.js'); // eslint-disable-line
 
-  return localeData.messages;
+  return localeData.default.messages;
```

Comment:
I'm not sure what has changed in recent react upgrade, and if that's best way to imports files. @mmahalwy please suggest the better solution ;)